ldms/src/test/Makefile.am

```diff
@@ -39,4 +39,17 @@ test_metric_SOURCES = test_metric.c
 test_metric_LDADD = -lldms
 test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
+install-data-hook: ldms-static-test-list.sh ldms-run-static-tests.sh ldms-static-test-bypass
```

Comment:
"ldms-static-test-list.sh ldms-run-static-tests.sh ldms-static-test-bypass" should not be listed as prerequisite targets, because they are build at configure time, not built by make targets.